validation/src/header.rs

```diff
@@ -13,6 +13,9 @@ pub enum ValidateHeaderError {
     /// Used when the timestamp in the header is lower than
     /// the median of timestamps of past 11 headers.
     HeaderIsOld,
+    /// Used when the timestamp in the header is more than 2 hours
+    /// from the current time.
+    HeaderIsMoreThan2HFromCurrentTime,
```

Comment:
Yes, I can do that. But I will change the name from `HeaderIsInFuture` to `HeaderIsTooFarInFuture` because it can be in the future and still pass the check, hence name could be misleading. :)

Reply:
Done.